client/extensions/woocommerce/woocommerce-services/lib/initialize-carrier-accounts-state.js

```diff
@@ -0,0 +1,41 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+export default () => {
+	const requiredFields = [
+		'account_number',
+		'name',
+		'address',
+		'city',
+		'state',
+		'state',
+		'country',
+		'postal_code',
+		'phone',
+		'email',
+		'company_name',
+		'job_title',
+		'company_website',
+		'license_agreement',
+	];
+
+	return {
+		UPS: {
+			settings: {
+				requiredFields,
+				values:{
+					country: 'US'
+				},
+				fieldErrors: {},
+				ignoreValidation: requiredFields.reduce( (accumulator, currentValue) => {   
+					accumulator[ currentValue ] = 'country' === currentValue ? false : true;
```

Comment:
Why do we ignore validations except `country`?

Reply:
This is only for the initial state. We're pre-setting the country to US, so we don't need to validate that field from the start. But if the field is updated by the user, `ignoreValidation.country` will be set to `false` again.